ts/Extensions/DataGrouping.ts

```diff
@@ -1166,6 +1288,31 @@ export default dataGrouping;
  * @apioption plotOptions.series.dataGrouping
  */
 
+/**
+ * Specifies how the points should be located on the x Axis inside the group.
+ * Points that are extremes can be set separately. Available options:
+ *
+ * - `start` places the point always at the beginning of the group
```

Comment:
" ... places the point always at the beginning ..." => Drop the "always". Also for the two next list items.